schema.sql

```diff
@@ -37,7 +37,7 @@ CREATE TABLE IF NOT EXISTS `players` (
   `posx` int NOT NULL DEFAULT '0',
   `posy` int NOT NULL DEFAULT '0',
   `posz` int NOT NULL DEFAULT '0',
-  `conditions` blob NOT NULL,
+  `conditions` blob DEFAULT NULL,
```

Comment:
This seems like a sane change, but considering there are already projects using that field, for example web AACs as string, it might be reasonable to do `DEFAULT ''`?